fragments/prj.conf.qemu_x86_osx

```diff
@@ -0,0 +1,2 @@
+CONFIG_RAM_SIZE=192
+CONFIG_ROM_SIZE=256
```

Comment:
In QEMU, we set CONFIG_XIP=Y, so I have to drop that, and increase the RAM_SIZE from 80 to 192, or I will run into the ram not fit error.